nwb_conversion_tools/datainterfaces/spikeglxdatainterface.py

```diff
@@ -9,16 +9,15 @@ class SpikeGLXRecordingInterface(BaseRecordingExtractorInterface):
 
     RX = SpikeGLXRecordingExtractor
 
-    def convert_data(self, nwbfile, metadata_dict: None, stub_test: bool = False, sync_with_ttl: bool = True,
-                     **conversion_options):
+    def convert_data(self, nwbfile, metadata_dict: None, stub_test: bool = False, sync_with_ttl: bool = True):
         """Primary function for converting recording extractor data to nwb."""
         if sync_with_ttl:
             ttl, states = self.recording_extractor.get_ttl_events()
             rising_times = ttl[states == 1]
             self.recording_extractor = SubRecordingExtractor(self.recording_extractor, start_frame=rising_times[0])
         super().convert_data(nwbfile=nwbfile, metadata_dict=metadata_dict, stub_test=stub_test)
 
-    def get_metadata():
+    def get_metadata(self):
         """Retrieve Ecephys metadata specific to the SpikeGLX format."""
         # TODO
-        raise NotImplementedError
+        return dict()
```

Comment:
You are better off just deferring this function to the `BaseRecordingExtractorInterface`